code/game/objects/structures/burrows.dm

```diff
@@ -639,12 +639,14 @@ percentage is a value in the range 0..1 that determines what portion of this mob
 /****************************
 	Burrow entering
 ****************************/
-/obj/structure/burrow/proc/enter_burrow(var/mob/living/L)
+/obj/structure/burrow/proc/enter_burrow(mob/living/L)
 	break_open()
 	spawn()
```

Comment:
missed a spawn

Reply:
though its important to understand the defualt value of spawn is 0, so this isnt actually sleeping as much as it just waiting for other code to execute